java/src/test/kotlin/auth/FakeClock.kt

```diff
@@ -0,0 +1,23 @@
+package auth
```

Comment:
`com.wgtwo.testing` ?

Reply:
or `com.wgtwo.testing.chrono`

Reply:
I can move it. Could have a been a util package for testing, I guess...